org.metaborg.spoofax.shell.eclipse/plugin.xml

```diff
@@ -13,4 +13,30 @@
            class="org.metaborg.spoofax.shell.client.eclipse.ReplView"/>
     </extension>
 
+    <!-- "Open file in REPL" command. -->
+    <extension point="org.eclipse.ui.commands">
+        <command id="org.metaborg.spoofax.shell.client.eclipse.handlers.open" name="Open in REPL"/>
+    </extension>
+    <extension point="org.eclipse.ui.handlers">
+        <handler commandId="org.metaborg.spoofax.shell.client.eclipse.handlers.open"
+                     class="org.metaborg.spoofax.shell.client.eclipse.handlers.OpenHandler"/>
+    </extension>
+
+    <extension point="org.eclipse.ui.menus">
+        <!-- Menu item in the project context menu (package- and project explorer). -->
+        <menuContribution locationURI="popup:org.metaborg.spoofax.eclipse.menu.project">
```

Comment:
This section is where I need to register the button in all the submenus of all the languages.